examples/gameroom/gameroom-app/src/store/api.ts

```diff
@@ -71,7 +70,8 @@ async function http(
       if (request.status >= 200 && request.status < 300) {
         resolve(request.response);
       } else {
-        console.error(request.response.message);
+        const responseText = JSON.parse(request.responseText);
```

Comment:
This should be called `responseBody`, as it is not text, but an object.

Reply:
Addressed